plugin.yaml

```diff
@@ -118,6 +118,51 @@ node_types:
       cloudify.interfaces.validation:
         creation: aws.ec2.securitygroup.creation_validation
 
+  cloudify.aws.nodes.Volume:
+    derived_from: cloudify.nodes.Volume
+    properties:
+      use_external_resource:
+        description: >
+          Indicate whether the resource exists or if Cloudify should create the resource.
+        type: boolean
+        default: false
+      resource_id:
+        description: >
+          Either the name or ID of the resource in Cloudify. If this is an existing
+          resource, you should provide the name or the ID of the resource in Amazon AWS.
+        type: string
+        default: ''
+      size:
+        description: >
+          The size of the volume in GB.
+        type: string
+      zone:
+        description: >
+          A string representing the AWS availability zone.
```

Comment:
if you want to mount this to existing instance it should be in the same availability zone